platform/viewer/cypress/support/commands.js

```diff
@@ -23,3 +25,17 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('openStudy', (patientName) => {
+    cy.visit('/');
+    cy.get('#patientName')
+      .type(patientName);
+    cy.get('.studylistStudy > .patientName')
+      .contains(patientName)
+      .click();
+  }
+);
+
+Cypress.Commands.add('drag', {prevSubject: 'element',},
```

Comment:
This is really cool! Some JSDoc `example` comments might be a nice value add here.

Reply:
I have added a usage example on the PR description.

**Usage:**
Drag and drop custom command: `cy.get('to-drag-element').drag('drop-element');